update_generic.sh

```diff
@@ -34,13 +34,13 @@ function upload_image() {
         ;;
     esac
     # No os property for cirros
-    _OS_PROPERTY=$([[ $DISTRO != 'cirros' ]] && echo " --property OS_DISTRO=$_OS_DISTRO" || echo "");
+    _OS_PROPERTY=$([[ $DISTRO != 'cirros' ]] && echo "--property os_distro=$_OS_DISTRO" || echo "");
     #common.sh
     sourcerc
     set +e
     OLD_ID="$(openstack image show "${IMAGE_RELEASE}" -f json | jq '.id' -r)"
     set -e
-    openstack image create "${IMAGE_RELEASE}" --file "${IMAGE_RELEASE}.img" --disk-format qcow2 --container-format bare --public --property hw_vif_multiqueue_enabled=true"$_OS_PROPERTY" --property hw_qemu_guest_agent='yes'
+    openstack image create "${IMAGE_RELEASE}" --file "${IMAGE_RELEASE}.img" --disk-format qcow2 --container-format bare --public --property hw_vif_multiqueue_enabled=true --property hw_qemu_guest_agent='yes' $_OS_PROPERTY
```

Comment:
include double quotes for the variable `"${_OS_PROPERTY]"`

Reply:
This is actually intended. In this case we want word splitting because it will otherwise not recognize it as an argument with a value.